pkg/config/proxy.go

```diff
@@ -365,6 +370,15 @@ func (cfg *BaseProxyConf) decorate(prefix string, name string, section *ini.Sect
 		}
 	}
 
+	if bandwidthMode, err := section.GetKey("bandwidth_limit_mode"); err == nil {
+		cfg.BandwidthLimitMode, err = NewBandwidthLimitMode(bandwidthMode.String())
```

Comment:
Validate it in `checkForCli` and `CheckForSvr`.

String value doesn't need a custom decode logic.

Reply:
moved validation of `bandwidth_limit_mode` to `CheckForCli`